Bottom: 0f26ec27acdb372474a865cefdfd630773dd70ee
Top:    8ac614d1ab50905142b17646fd000c210d52efb9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 17:01:50 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
